RR_manuscript/supplementary_materials.Rmd

```diff
@@ -77,7 +94,7 @@ msa_models_unique <- left_join(
   by = "animal"
 ) %>% 
   left_join(., 
-    read_csv("./data/questionnaires/msa_intake_form_untidy.csv") %>%
+    read_csv(here("./data/questionnaires/msa_intake_form_untidy.csv")) %>%
```

Comment:
no need for here()

Reply:
Same as above.

Reply:
Same as above

Reply:
Same as... lol jk. Ill make changes tomorrow morning when I have a minute. Pesado.